themes/openy_themes/openy_lily/openy_lily.theme

```diff
@@ -879,3 +879,16 @@ function openy_lily_preprocess_field__node__field_location_directions(array &$va
   // 'Separate link text and URL' format by default.
   $variables['items'][0]['content']['#url_title'] = $link_title;
 }
+
+/**
+ * Preprocess paragraph - social list.
+ *
+ * @param $variables
+ */
+function openy_rose_preprocess_paragraph_social_list(&$variables) {
+  $block = $variables['content']['field_prgrf_sl_block'];
+  if (isset($block[0]['#name']) && isset($block[0]['#display_id'])) {
```

Comment:
What happens if there are 2+ blocks on a page?

Reply:
We have 2 blocks as expected. I checked.